Gemfile.lock

```diff
@@ -198,10 +203,18 @@ GEM
       i18n (>= 0.7)
       multi_json
       request_store (>= 1.0)
-    google-protobuf (3.25.3-aarch64-linux)
-    google-protobuf (3.25.3-arm64-darwin)
-    google-protobuf (3.25.3-x86_64-darwin)
-    google-protobuf (3.25.3-x86_64-linux)
+    google-protobuf (4.27.2-aarch64-linux)
```

Comment:
Some sort of breaking change between protobuf 25.3 and 27.2 in the Ruby bindings: https://github.com/protocolbuffers/protobuf/tree/main/ruby#version-number-scheme

* https://github.com/protocolbuffers/protobuf/releases/tag/v26.0 (<- [breaking changes announcement](https://protobuf.dev/news/2023-12-27/#ruby-breaking-changes), sass-embedded is our only dep using it and was updated to be compatible)
* https://github.com/protocolbuffers/protobuf/releases/tag/v27.0